### DISCUSSION TITLE
Linking documents that belong to each other

### DISCUSSION THREAD

**Discussion by zjean** [jonaswinkler/paperless-ng#386] [Feature Requests]
Hi guys

First of all: thanks @jonaswinkler for your awesome job in creating Paperless NG!

I was wondering about the following: What is a good way to link documents to each other. For instance, documents that belong to the same e-mail message.
I might have received a message from the tax office, about the payment of taxes that consists of multiple pdf files. One with an explanation of this year's regulations, and one with the actual payment that I have to make.
Or I get the proforma invoice of a company (document 1), together with an explanation of their services (document 2).
These documents belong to each other, since they might reference one another, and are from the same correspondent.
However, creating a tag to group them, feels like overkill, since it would be a one-shot tag.

How are others handling this scenario? Or am I organizing too much / expecting too much from the metadata side for which I should use the regular search and filtering tools?

**Comment by shamoon** [comment]
Yea its something I've thought about too. I came from another DMS (Teedy) which has a different conception of documents such that one document can have many files associated with it. At first I sorta missed that and other systems have a "related documents" field which I suppose could be implemented here but realistically I have found that most of the time I can "get" to the right document without too much effort making that kind of feature less important at least for me. E.g. in your example the two will presumably have the same correspondent and similar / same dates, so find one, click the correspondent to filter for just that correspondent and if sorted by date the other document will be right next to it. Similarly if they have the same tag. Of course in your example you could also merge the PDFs into one doc...

But yea, "related" is a thought.

**Comment by skorvek** [comment]
Seems to me like tags themselves are sufficient along with the correspondent and document type.
The same correspondent relates them. Then use a combination of tags to filter further. Unpaid. Tax Year XXXX. Invoice and Instructions as separate doc types.  Maybe "complete" or "archived" (and definitely "paid" for paid invoices.
Then set up views for what you want to get to quickly. 

simply tax year and correspondent should cover this use case I'd think. Then you'd see the invoice and instructions.
